src/ConcealWallet/ConcealWallet.cpp

```diff
@@ -1155,10 +828,10 @@ bool simple_wallet::reset(const std::vector<std::string> &args) {
   return true;
 }
 
-bool simple_wallet::get_reserve_proof(const std::vector<std::string> &args)
+bool conceal_wallet::get_reserve_proof(const std::vector<std::string> &args)
 {
 	if (args.size() != 1 && args.size() != 2) {
-		fail_msg_writer() << "Usage: get_reserve_proof (all|<amount>) [<message>]";
+		fail_msg_writer() << "Usage: balance_proof (all|<amount>) [<message>]";
```

Comment:
Could we also rename the file generated to `balance_proof` so it is less confusing to users?